docs/evaluation/how_to_guides/llm_as_judge.mdx

```diff
@@ -1,24 +1,36 @@
+
 import {
   CodeTabs,
   python,
   typescript,
 } from "@site/src/components/InstructionsWithCode";
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import '@site/src/css/custom.css';
+
+
 # How to define an LLM-as-a-judge evaluator
 
 :::info Key concepts
+- [LLM-as-a-judge evaluator](/evaluation/concepts#llm-as-judge)
+:::
 
-- [LLM-as-a-judge evaluator](../concepts#llm-as-judge)
+LLM applications can be challenging to evaluate since they often generate conversational text with no single correct answer.
 
-:::
+This guide shows you how to define a LLM-as-a-judge evaluator for [offline evaluation](/evaluation/concepts#offline-evaluation) using either the LangSmith SDK or the UI. Note: To run evaluations in real-time on your production traces, refer to [setting up online evaluations](/observability/how_to_guides/online_evaluations#configure-llm-as-judge-evaluators). 
+
+<Tabs className="interface-tabs">
```

Comment:
the tabs look kinda funky, imo better to just have separate headers

Reply:
updating styling slightly but def could use improvement

content becomes quite long if we dont tab it. since its two very diff group of users i think we should keep content separated for easier readability